src/ALCProxy.Proxy/ProxyBuilder.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.Loader;
+using Microsoft.VisualBasic.CompilerServices;
+
+namespace ALCProxy
+{
+
+    public interface ITest
+    {
+        public void DoThing();
+
+        public void DoThing2(int a, List<string> list, Test2 t);
+    }
+    public class Test2 { }
+
+    public class Test : ITest
```

Comment:
What's our plan for testing this? Is the plan to do manual testing (not triggered by CI) for now and work on that going forward? What we have currently won't get triggered in CI (at least not without some infra work to enable that in the repo). I am asking just to understand what we want to do here and corefxlab needs to accommodate some testing scenario or if we can rely on the existing regular library + unit test project structure.